tests/lib/prepare.sh

```diff
@@ -130,12 +130,6 @@ update_core_snap_for_classic_reexec() {
     rm squashfs-root/usr/lib/snapd/* squashfs-root/usr/bin/snap
     # and copy in the current libexec
     cp -a "$LIBEXECDIR"/snapd/* squashfs-root/usr/lib/snapd/
-    case "$SPREAD_SYSTEM" in
-        fedora-*|centos-*|amazon-*)
-            # RPM can rewrite shebang to #!/usr/bin/sh
-            sed -i -e '1 s;#!/usr/bin/sh;#!/bin/sh;' squashfs-root/usr/lib/snapd/snap-device-helper
```

Comment:
For some context. The reason this was here in the first place is that we rebuild snapd on, say Fedora 31, and repackage the 16.04-based core snap with the package built on fedora. This now puts the `/usr/bin/sh` shebang line inside the core snap, which doesn't exist there.

We fixed this with cc4e91c006d so the workaround is no longer necessary.

Reply:
@bboozzoo can you please include this in the commit message ^

Reply:
I'll update the commit message and force push